source/_integrations/imap.markdown

```diff
@@ -93,7 +93,7 @@ sensor:
     password: YOUR_PASSWORD
     search: FROM <sender@email.com>, SUBJECT <subject here>
     # Or use X-GM-RAW search-term like this, to find unread emails from the last 7 days in your inbox
-    # search: "X-GM-RAW 'in: inbox newer_than:7d is:unread'"
+    # search: "X-GM-RAW \"in: inbox newer_than:7d is:unread\""
```

Comment:
Maybe change the quotes on our end to remove the need for escaping entirely?
```suggestion
    # search: 'X-GM-RAW "in: inbox newer_than:7d is:unread"'
```

Reply:
Sure, if we can ignore the [standards](https://developers.home-assistant.io/docs/documenting/standards/#single-vs-double-quotation-marks), it's totally fine with me. That's the only reason why i ended up using " \" :-)

Reply:
Yeah, I think it is fine in this case.